toolkit/tools/imagecustomizer/docs/building-packages.md

```diff
@@ -1,97 +0,0 @@
-# Building custom packages
```

Comment:
Why was this file deleted?

Reply:
This is distro focused, it should live in azl repo IMO

Reply:
I created this doc to be a simplified way of building packages. The AZL repo’s instructions use the toolkit/toolchain, which heavily complicates things.

Please restore this file.